samples/debug-utils/logging_macros.h

```diff
@@ -18,7 +18,7 @@
 #define __SAMPLE_ANDROID_DEBUG_H__
 #include <android/log.h>
 
-#if 1
+#if OBOE_ENABLE_LOGGING
```

Comment:
Why is this change here. It conflicts with an older fix.

Reply:
this is for this: https://github.com/google/oboe/issues/175. 

add a flag to guard it, then could use compile flag to control it is on or off